### PR TITLE
Show error if dashboard name is empty or space

### LINKED ISSUE #212: Dashboard save : do not allow empty string or space as dashboard name
# Description
Currently when new dashboard is created , or old dashboard is updated, we can enter space as dashboard name. This should not be allowed.

On UI side, before calling api to create/update dashboard,  trim dashboard name , and check if it is empty, show error / popup is dashboard name is empty.

### PR DESCRIPTION
**Pull request by PratikshaPatil77**
# Description
The user will not be able to input space when creating a new dashboard or updating an existing one since the dashboard name cannot be empty.When creating or editing a dashboard name, an error warning will appear if the user attempts to enter the dashboard name as a space.

Fixes #212 

# Testing
Describe how you tested this code. How can the reviewers reproduce your tests?

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [ ] I have self-reviewed this PR.
- [ ] I have removed all print-debugging and commented-out code that should not be merged.
- [ ] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [ ] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: static/js/dashboard.js [hunk 1]
```diff
@@ -1130,9 +1130,22 @@ function addDbSettingsEventListeners() {
 }
 
 function saveDbSetting() {
+    let trimmedDbName = $('.dbSet-dbName').val().trim();
+    
+    if (!trimmedDbName) {
+        // Show error message or handle the case where the name is empty
+        alert('Dashboard name cannot be empty or consist of only spaces.');
```
Comment (sunitakawane) [inline static/js/dashboard.js:1137]: change the message to  `Dashboard name cannot be empty`

Comment (PratikshaPatil77) [reply static/js/dashboard.js:1137]: Ok so we don't have to mention about white-spaces in message.

**Comment by sunitakawane** [comment]
@sonamgupta21 can you verify this

**Comment by sonamgupta21** [comment]
@sunitakawane Do we want a popup or just an error-tip below the input box?
image

**Review by sonamgupta21 — APPROVED** [review]

**Pull request merged**